src/control/XournalMain.cpp

```diff
@@ -302,6 +302,15 @@ int XournalMain::run(int argc, char* argv[])
 
 	Control* control = new Control(gladePath);
 
+	if (control->getSettings()->isDarkTheme())
+	{
+		string icon = gladePath->getFirstSearchPath() + "/iconsDark/";
+		gtk_icon_theme_prepend_search_path(gtk_icon_theme_get_default(), icon.c_str());
+	}
+
+	string icon = gladePath->getFirstSearchPath() + "/icons/";
```

Comment:
Hi @andreasb242 ! I don't know if this is the line which causes this, but now "dark-themed" icons doesn't get loaded anymore, did you make it available only on "make install" installation?

Reply:
I switched to GTK Icon loading. They are still installed within the ui Folder, which should be installed.
And I also tested it, but one thing I was not sure:
gtk_icon_theme_prepend_search_path or gtk_icon_theme_apend_search_path

After a few tries this solution works for me.

Reply:
![image](https://user-images.githubusercontent.com/20294254/58508048-6703f900-8193-11e9-85f9-2bdc5825ea3d.png)
Hey there, I'm not sure what happened, but I've just installed xournalpp on a fresh new system and dark icons don't seem to get loaded. Where can I start digging?